guidemaker-ember-template/src/styles/addon.css

```
@@ -224,3 +224,19 @@ h5 > a:hover > svg,
 h6 > a:hover > svg {
   fill: var(--color-brand);
 }
+
+
+/* table styles  */
+table {
+  width: 100%;
+  text-align: left;
+  border-collapse: collapse;
+}
+thead th{
+  background-color: var(--color-gray-200);
+}
+td, th {
+  border: 1px solid var(--color-gray-300);
+  padding: 4px;
```

We have spacing variables as well. Would use the smallest one here, which is a bit larger but looks good.

```suggestion
  padding: var(--spacing-1);
```